test/misc.jl

```diff
@@ -804,3 +804,9 @@ end
 
     GC.safepoint()
 end
+
+@testset "traits" begin
+    @test Base.OrderStyle(1) isa Base.Ordered
+    @test Base.ArithmeticStyle(1) isa Base.ArithmeticWraps
+    @test Base.RangeStepStyle(1:2) isa Base.RangeStepRegular
+end
```

Comment:
These seem awfully specific to me, especially since these are not exported. If these are not covered, it must mean there is something higher level that's not being tested, or perhaps coverage is missing them due to always being inlined?

Reply:
Agreed. The type version of the traits is tested, but just not the instance version. I guess the instance methods are offered for convenience? But since they aren't exported or used, maybe we could just delete them?

Reply:
Does that make sense?